.github/workflows/kubernetes_test.yaml

```diff
@@ -51,65 +54,54 @@ jobs:
         run: |
           conda install -c anaconda pip
           pip install .[dev]
-      - name: Download and Install Minikube and Kubectl
+      - name: Download and Install Kind and Kubectl
         run: |
           mkdir -p bin
           pushd bin
-          curl -L https://github.com/kubernetes/minikube/releases/download/v1.22.0/minikube-linux-amd64 -o minikube
-          chmod +x minikube
+          curl -L https://github.com/kubernetes-sigs/kind/releases/download/v0.12.0/kind-linux-amd64 -o kind
+          chmod +x kind
 
           curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.19.0/bin/linux/amd64/kubectl
           chmod +x kubectl
 
           echo "$PWD" >> $GITHUB_PATH
           popd
-      - name: Start Minikube
+      - name: Enable docker permissions for user
         run: |
           sudo docker ps
           sudo usermod -aG docker $USER && newgrp docker
-          minikube start --kubernetes-version=1.19.4 --driver=docker --cpus 4 --memory 12288 --wait=all
-      - name: Print minikube and kubectl versions
+
+          docker info
+          docker ps
+      - name: Pull docker images
         run: |
-          minikube version
-          kubectl version
-      - name: Use minikube docker daemon
+          images=${{ env.image_names }}
```

Comment:
I think this is double work. We should remove all the `docker pull` and `kind load` since kubernetes will pull as needed